test/case/infix_dhcp/dhcp_routes/test.py

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python3
-"""
-DHCP option 121 vs option 3
+"""DHCP option 121 vs option 3
 
-Verify DHCP option 121 (static routes) is used over option 3 and that the
-routes exist in the operational datastore.
+Verify that DHCP option 121 (classless static routes) is used over the
+older option 3 (default gateway) in the DHCP client, see RFC3442.  Use
+the routing RIB in the operational datastore to verify.
 
-Installing unrelated routes from a DHCP server should not affect already
-existing routes.  To verify this a canary route is set up in the client
-before initiating DHCP.  This canary route does not need to be reachable
-before a DHCP lease has been acquired.
+Installing routes from a DHCP server should not affect already existing
+(static) routes.  To verify this, a static canary route is set in the
```

Comment:
Perhaps change
_static canary route is_
to
_static canary route (20.0.0.0/24) is_

Reply:
> ... a static canary route (20.0.0.0/24 via 192.168.0.2) is ...

because the nh differs as well (otherwise I wouldn't have been able to read this old code myself =)